modules/media/php/media.class.inc

```diff
@@ -108,31 +109,23 @@ class Media extends \NDB_Menu_Filter
             $languageList[$language] = $language;
         }
 
-        // Form Elements
-        $this->addBasicText('pSCID', 'PSCID', ["size" => 9, "maxlength" => 7]);
-        $this->addBasicText('fileName', 'File Name');
-        $this->addSelect('fileType', 'File type', $fileTypeList);
-        $this->addSelect('visitLabel', 'Visit Label', $visitList);
-        $this->addSelect('site', 'For Site', $siteList);
-        $this->addBasicText('uploadedBy', 'Uploaded By');
-        $this->addSelect('instrument', 'Instrument', $instrumentList);
-        $this->addSelect('language', 'Language', $languageList);
-
-        // Add hidden files filter is user is admin
+        $hiddenOptions = [];
         if ($this->hasHidePermission) {
-            // Unable to create checkbox using addCheckbox(), using addSelect for
-            // the time being
-            // select options
             $hiddenOptions = array(
-                              ""  => 'Both',
-                              "0" => 'Visible only',
-                              "1" => 'Hidden only',
+                              0 => 'Visible only',
```

Comment:
To display both, one needs to clear the filters?

Reply:
Clear the filter or return it the blank value that already exists by default in the select dropdown.